homeassistant/components/whirlpool/sensor.py

```diff
@@ -205,11 +203,6 @@ def __init__(
         """Initialize the washer sensor."""
         self._wd: WasherDryer = washer_dryer
 
-        if washer_dryer.name == "dryer":
```

Comment:
This timestamp sensor should have a fixed "clock" icon and not the same icon as the device's main state sensor.
(reviewing the implementation of this Sensor class is also on my todo list, since it deviates from the rest of the integration)